sdk/servicebus/service-bus/samples/typescript/gettingStarted/atomManagementApi.ts

```diff
@@ -0,0 +1,32 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+"use strict";
+
+import { ServiceBusAtomManagementClient, HttpOperationResponse } from "@azure/service-bus";
+
+async function main(): Promise<void> {
+  const serviceBusAtomManagementClient: ServiceBusAtomManagementClient = new ServiceBusAtomManagementClient(
+    "<insert-connection-string>"
+  );
+
+  const response: HttpOperationResponse = await serviceBusAtomManagementClient.createQueue(
+    "testQueuePath4",
+    {
+      LockDuration: "PT1M",
+      MaxSizeInMegabytes: "1024",
+      RequiresDuplicateDetection: "false",
+      RequiresSession: "false",
+      DeadLetteringOnMessageExpiration: "false",
+      MaxDeliveryCount: "10",
+      EnableBatchedOperations: "true",
+      EnablePartitioning: "false"
+    }
+  );
```

Comment:
Is it mandatory to provide all of the above to create the queue? If not, I would pick one or two most commonly used ones and showcase that in the sample. For example, `EnablePartitioning` and `RequiresSession`

Also, I believe it would be a good user experience if we can tell a story here.
- Use `createQueue` to create a queue with partitioning and sessions enabled
- Use `getQueue` to show kind of information can the user expect to learn from a queue
- Use `updateQueue` to update maybe the MaxDeliveryCount
- Use `listQueues`, followed by `deleteQueue`

Then we can have a description up top (like we do for other samples) that can say something like, "Below are the management operations that you can do with queues. Similar support exists for topics and subscriptions"

Reply:
Agree with improving sample - working on updating this.

Other than this, should we consider returning only the "parsed response" to the user? This is so they don't have to worry about the different fields on `HttpOperationResponse` and it's structure which may change based on core-http updates?

However, user could (should?) alternatively be able to pass in onUpload/onDownload callbacks as part of the HTTP request. Since this is something users were able to do so previously, by passing in 'chunkedCallback'.

cc @chradek @bterlson

Reply:
I dont follow the onUpload/onDownload comment. What is the user scenario for a "chunked callback"?

Reply:
Can ignore the chunked callback point, it was related to an internal and doesn't correspond to a user scenario in the older SDK as such.